tests/realtikvtest/pessimistictest/pessimistic_test.go

```diff
@@ -3091,6 +3095,9 @@ func TestPessimisticAutoCommitTxn(t *testing.T) {
 	rows = tk.MustQuery("explain update t set i = -i").Rows()
 	explain = fmt.Sprintf("%v", rows[1])
 	require.Regexp(t, ".*SelectLock.*", explain)
+	rows = tk.MustQuery("explain update t set i = -i where i = -1").Rows()
```

Comment:
We could also cover the batch point plans.

Reply:
done